app/resources/category_resource.rb

```diff
@@ -0,0 +1,9 @@
+class CategoryResource < BaseResource
```

Comment:
Can we also add a filter by slug, and unlimited pagination?
```
require 'unlimited_paginator`
...
paginator :unlimited
```
`filters :parent_id, :slug`